gammapy/maps/core.py

```diff
@@ -87,6 +87,10 @@ def geom(self):
         """Map geometry as a `~gammapy.maps.Geom` object."""
         return self._geom
 
+    @property
+    def single_spatial_bin(self):
```

Comment:
```suggestion
    def has_single_spatial_bin(self):
```
But shouldn't this live on `Geom` instead?